content/en/docs/Creating derivatives/Packer/build_azure.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Build Azure images"
+linkTitle: "Build Azure images"
+weight: 4
+date: 2017-01-05
+description: >
+  This section documents the procedure to deploy cOS (or derivatives) images
+  in Azure public cloud provider by using the cOS Vanilla image.
+---
+
+![](https://docs.google.com/drawings/d/e/2PACX-1vSqJWcFThP7K2HS551LCqs73l4ZncXElLjlbCvxY96Ga2Jbjnq79j-DEjaccUZvYEQyphWiDQc9flxk/pub?w=1223&h=691)
+
+Requirements:
+
+* Packer
+* Azure access keys with the appropriate roles and permissions
+* [A Vanilla image](../../../development/build_raw_images#azure)
```

Comment:
```suggestion
* [A Vanilla image](../../../getting-started/booting/#importing-an-azure-image-manually)
```

I would avoid to link to dev docs, as this is mostly user facing, no need to build cOS to get the Azure image anymore :)

Reply:
wow, totally forgot that existed, was even thinking of creating docs on how to upload all the base cloud vanilla images :o